### DISCUSSION TITLE
Feedback & Suggestions

### DISCUSSION THREAD

**Discussion by officiallymarky** [louislam/dockge#131] [Feature Request]
Another homerun, this is a great tool.

Some suggestions I have.

* It gets a bit crowded, would be nice to be able to slide the left column when editing
* dockge compose when stored with other stacks confuses dockge
* Maybe even allow middle column to shrink when editing, or better yet have an "edit mode" that gives you more real estate
* respect new lines, I like to have a new line between each service  and global settings like networks:, when I view my compose in dockge, all my new lines are gone.
* 2fa would be nice
* not sure how you would do it, but would be nice to have versioning for the compose files some how, if it is done globally, or per stack folder with git, or something.
* terminal only seems to use about 60% of it's space
* would be nice to expand terminal so you have some more width to read as needed
* file for a new icon for dashboard icons: https://github.com/walkxcode/dashboard-icons

Space is my big concern, it's just really crammed, even an ultra-wide.

**Comment by Z1mDMan** [comment]
I agree with OP's suggestions and also having the ability to edit more than just the YAML files would be nice. Some of my containers use JSON files for configurations and that would help simplify making changes to those for me without having to leave Dockge.  Thanks again for making Dockge!

**Comment by e-labInnovations** [comment]
Any update on 2FA?
